imagetest/test_suites/gveperf/startupscripts/netserver_startup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# This script installs iperf on a VM and starts an iperf server for the client
+# to test the network bandwidth between the two VMs.
+
+sleepduration=120
+
+if [[ -f /usr/bin/apt ]]; then
+  echo "apt found Installing iperf."
+  sudo apt update && sudo apt install iperf
+elif [[ -f /bin/apt-get ]]; then
+  echo "apt found Installing iperf."
+  sudo apt-get update && sudo apt-get install iperf
```

Comment:
apt and apt-get are functionally the same from this point of view, we can remove the apt-get case (it will never be called).

Reply:
Done.